src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Jaeger\Thrift\Span as JTSpan;
+use Jaeger\Thrift\Tag;
+use Jaeger\Thrift\TagType;
+use OpenTelemetry\SDK\Trace\SpanDataInterface;
+use RuntimeException;
+
+class SpanConverter
+{
+    const STATUS_CODE_TAG_KEY = 'op.status_code';
+    const STATUS_DESCRIPTION_TAG_KEY = 'op.status_description';
+
+    /**
+    * Convert span to Jaeger Thrift Span format
+    */
+    public function convert(SpanDataInterface $span): JTSpan
+    {
+        $references = $tags = $logs = [];
+        $startTime = (int) ($span->getStartEpochNanos() / 1e3); // microseconds
```

Comment:
`AbstractClock::nanosToMicro()` does this

Reply:
Switched!